CONTRIBUTING.md

```diff
@@ -42,29 +42,13 @@ Or run individual suites using:
 
 ```shell
 $ make test-unit
-$ make test-functional
+$ make test-integration
+$ make test-matrix
 ```
 
-You can get a coverage report in text:
+You can get a coverage report in text and HTML by running:
 
 ```shell
 $ make test-coverage
```

Comment:
This resolves to `vendor/bin/phpunit --coverage-text`. Will that include HTML? Where does it get written to?